testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -12230,6 +12235,8 @@ void testTimestampDiff(boolean coercionEnabled) {
     f.checkScalar("time_trunc(time '12:34:56', hour)",
         "12:00:00", "TIME(0) NOT NULL");
     f.checkNull("time_trunc(cast(null as time), second)");
+    f.checkNull("time_trunc(cast(null as time), minute)");
```

Comment:
I don't really see how these tests help us. We already have a `checkNull` test for `second`, the value of the 2nd operand isn't going to influence the behavior (as long as it is the same type).

Reply:
I just tested and it seems that the second parameter is null is not allowed, which will fail to parse

Reply:
thanks